### PR TITLE
comment out all the fetching context related code

### PR DESCRIPTION
**Pull request by enriquepablo**
comment out all the fetching context related code

### CODE REVIEW

File: plugins/eduid_action.mfa/component.js [hunk 1]
```diff
@@ -4,7 +4,6 @@ import { connect } from "react-redux";
 
 import i18n from "i18n-messages";
 import {
-  appFetching,
```
Comment (njons) [inline plugins/eduid_action.mfa/component.js]: I just want to double check that it's ok to go beyond commenting this out here and fully remove it

File: src/actions/ActionWrapper.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
 export const APP_LOADED = "APP_LOADED";
 export const APP_LOADING = "APP_LOADING";
-export const APP_FETCHING = "APP_FETCHING";
```
Comment (njons) [inline src/actions/ActionWrapper.js]: I just want to double check that it's ok to go beyond commenting this out here and fully remove it

File: src/actions/SignupMain.js [hunk 3]
```diff
@@ -2,7 +2,6 @@ export const GET_CODE_STATUS = "GET_CODE_STATUS";
 export const GET_CODE_STATUS_FAIL = "GET_CODE_STATUS_FAIL";
 export const APP_LOADED = "APP_LOADED";
 export const APP_LOADING = "APP_LOADING";
-export const APP_FETCHING = "APP_FETCHING";
```
Comment (njons) [inline src/actions/SignupMain.js]: I just want to double check that it's ok to go beyond commenting this out here and fully remove it

File: src/components/ActionWrapper.js [hunk 4]
```diff
@@ -59,14 +60,14 @@ class ActionWrapper extends Component {
...
-      </FetchingContext.Provider>
+      {/*</FetchingContext.Provider>*/}
+      </div>
     );
   }
 }
 
 ActionWrapper.propTypes = {
   redirect: PropTypes.string,
-  is_fetching: PropTypes.bool
```
Comment (njons) [inline src/components/ActionWrapper.js]: I just want to double check that it's ok to go beyond commenting this out here and fully remove it

File: src/containers/ActionWrapper.js [hunk 5]
```diff
@@ -7,7 +7,6 @@ import * as actions from "actions/ActionWrapper";
 const mapStateToProps = (state, props) => {
   return {
     redirect: state.config.redirect,
-    is_fetching: state.config.is_fetching
```
Comment (njons) [inline src/containers/ActionWrapper.js]: I just want to double check that it's ok to go beyond commenting this out here and fully remove it

File: src/containers/SignupMain.js [hunk 6]
```diff
@@ -6,7 +6,6 @@ import * as actions from "actions/SignupMain";
 
 const mapStateToProps = (state, props) => {
   return {
-    is_fetching: state.config.is_fetching,
```
Comment (njons) [inline src/containers/SignupMain.js]: I just want to double check that it's ok to go beyond commenting this out here and fully remove it

File: src/reducers/ActionWrapper.js [hunk 7]
```diff
@@ -22,10 +22,6 @@ let actionWrapperReducer = (state = configData, action) => {
         ...state,
         is_app_loaded: true
       };
-    case actions.APP_FETCHING:
```
Comment (njons) [inline src/reducers/ActionWrapper.js]: I just want to double check that it's ok to go beyond commenting this out here and fully remove it

File: src/reducers/ActionWrapper.js [hunk 8]
```diff
@@ -22,10 +22,6 @@ let actionWrapperReducer = (state = configData, action) => {
...
-      return {
-        ...state,
-        is_fetching: true
```
Comment (njons) [inline src/reducers/ActionWrapper.js]: I just want to double check that it's ok to go beyond commenting this out here and fully remove it

File: src/reducers/SignupMain.js [hunk 9]
```diff
@@ -13,124 +13,112 @@ const configData = {
...
+//const unFetchingActions = [
+  //actions.APP_LOADED,
+//];
 
 let signupReducer = (state = configData, action) => {
   switch (action.type) {
     case actions.APP_LOADING:
       return {
         ...state,
-        is_fetching: true,
```
Comment (njons) [inline src/reducers/SignupMain.js]: I just want to double check that it's ok to go beyond commenting this out here and fully remove it

File: src/reducers/SignupMain.js [hunk 10]
```diff
@@ -13,124 +13,112 @@ const configData = {
...
         is_app_loaded: false
       };
     case actions.APP_LOADED:
       return {
         ...state,
-        is_fetching: false,
```
Comment (njons) [inline src/reducers/SignupMain.js]: I just want to double check that it's ok to go beyond commenting this out here and fully remove it

File: src/reducers/SignupMain.js [hunk 11]
```diff
@@ -13,124 +13,112 @@ const configData = {
...
         is_app_loaded: true
       };
-    case actions.APP_FETCHING:
```
Comment (njons) [inline src/reducers/SignupMain.js]: I just want to double check that it's ok to go beyond commenting this out here and fully remove it

File: src/reducers/SignupMain.js [hunk 12]
```diff
@@ -13,124 +13,112 @@ const configData = {
...
-      return {
-        ...state,
-        is_fetching: true
-      };
     case actions.GET_CODE_STATUS:
       return {
         ...state,
         ...action.payload,
-        is_fetching: true
```
Comment (njons) [inline src/reducers/SignupMain.js]: I just want to double check that it's ok to go beyond commenting this out here and fully remove it

File: plugins/eduid_action.mfa/component.js (hunk 1 above)
Comment (enriquepablo) [reply plugins/eduid_action.mfa/component.js]: Yes all are consciously removed. In particular: The appFetching action was not being used anywhere anymore

File: src/actions/ActionWrapper.js (hunk 2 above)
Comment (enriquepablo) [reply src/actions/ActionWrapper.js]: Since the action is not used, the action type is neither

File: src/actions/SignupMain.js (hunk 3 above)
Comment (enriquepablo) [reply src/actions/SignupMain.js]: Since the action is not used, the action type is neither

File: src/components/ActionWrapper.js (hunk 4 above)
Comment (enriquepablo) [reply src/components/ActionWrapper.js]: The actionWrapper does not need to know whether the app is fetching any more; just the buttons need to know

File: src/containers/ActionWrapper.js (hunk 5 above)
Comment (enriquepablo) [reply src/containers/ActionWrapper.js]: The actionWrapper does not need to know whether the app is fetching any more; just the buttons need to know

File: src/containers/SignupMain.js (hunk 6 above)
Comment (enriquepablo) [reply src/containers/SignupMain.js]: The signup app does not need to know whether the app is fetching any more; just the buttons need to know

File: src/reducers/ActionWrapper.js (hunk 8 above)
Comment (enriquepablo) [reply src/reducers/ActionWrapper.js]: The appFetching action is being used no more

File: src/reducers/SignupMain.js (hunk 9 above)
Comment (enriquepablo) [reply src/reducers/SignupMain.js]: The appLoading action is removed in another PR

File: src/reducers/SignupMain.js (hunk 11 above)
Comment (enriquepablo) [reply src/reducers/SignupMain.js]: the action has been removed

File: src/reducers/SignupMain.js (hunk 10 above)
Comment (enriquepablo) [reply src/reducers/SignupMain.js]: is_fetching is set to false by another action

**Pull request merged**